From 95bc3a89353211d19407de338fb3a3f02389394b Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Mon, 15 May 2017 16:05:00 +0800
Subject: [PATCH] Revert "arm64: rockchip_defconfig: enable CONFIG_IKCONFIG"

This reverts commit 28f4152aad7fc402f6f545f07f4193aac7a05087.

For pass Android 7.1 CTS:
android.security.cts.KernelSettingsTest#testNoConfigGz

Change-Id: I9bd7d4c6c06b7d43fa51a7f02eecca8cedc61c1d
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 305f14c7020c..edc020547afe 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -8,8 +8,6 @@ CONFIG_TASKSTATS=y
 CONFIG_TASK_DELAY_ACCT=y
 CONFIG_TASK_XACCT=y
 CONFIG_TASK_IO_ACCOUNTING=y
-CONFIG_IKCONFIG=y
-CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=19
 CONFIG_CGROUPS=y
 CONFIG_CGROUP_DEBUG=y
-- 
2.35.3

